tools/iothub-explorer/iothub-explorer.js

```diff
@@ -275,6 +297,22 @@ else {
   usage();
 }
 
+function createDevice(deviceId){
```

Comment:
Can you update the 'create' command to call into this also, so we don't add redundant code to the file?  I think your createDevice function will need to be able to take either a deviceId or a JSON object (with a deviceId property) as an argument, but with that little modification it should work nicely for both commands.